### PR TITLE
Add nextjs examples

### PR DESCRIPTION
**Pull request by brunos3d**
- feat: add base project structure for angular
- feat: update project name structure
- feat: add base project structure for qwik

### CODE REVIEW

**Comment by ScriptedAlchemy** [comment]
In the future, we should be able to use PluginSystem instance to replace all delegate module responsibilities. 

https://github.com/module-federation/universe/blob/53403a66b96ba170bd11258eeeaf80aa8addf6e5/packages/runtime/src/utils/hooks/pluginSystem.ts

I will send a PR soon

**Review by ScriptedAlchemy — APPROVED** [review]

File: examples/nextjs-13/template/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "lint": "concurrently \"cd apps/host && npm run lint\" \"cd apps/remote && npm run lint\""
   },
   "dependencies": {
-    "@module-federation/dashboard-plugin": "^2.8.0-beta.7",
+    "@module-federation/dashboard-plugin": "^2.8.0-beta.6",
```
Comment (zackarychapple) [inline examples/nextjs-13/template/package.json:13]: why did this get bumped back down?

Comment (brunos3d) [reply examples/nextjs-13/template/package.json:13]: I need to release a new version with a fix that removes the skipPost from FederationDashboardPlugin and keep it only for NextFederationPlugin

Comment (brunos3d) [reply examples/nextjs-13/template/package.json:13]: It was causing an issue with the post dashboard data

**Pull request merged**